zio-cache/shared/src/main/scala/zio/cache/ScopedCacheImplementation.scala

```diff
@@ -123,52 +124,53 @@ private[cache] class ScopedCacheImplementation[Key, Environment, Error, Value](
               ensureMapSizeNotExceeded(key) *> scoped
             case complete @ MapValue.Complete(key, _, _, _, timeToLive) =>
               trackHit()
-              if (hasExpired(timeToLive)) {
+              if (now.isAfter(timeToLive)) {
```

Comment:
@strokyl I think we're loosing a piece of information here. The code was super explicit before: `hasExpired(timeToLive)`

What do you think of doing this to keep the explicitness of what this code is testing:
```suggestion
              val hasExpired = now.isAfter(timeToLive)
              if (hasExpired) {
```